README.md

```diff
@@ -56,24 +56,24 @@ to obtain the script.
 We also provide the docker image which installed cuttlefish debian packages
 inside; including `cuttlefish-base`, `cuttlefish-user`, and
 `cuttlefish-orchestration`.
-Currently it's available for ARM64 architecture.
+Currently it's available for x86_64 and ARM64 architecture.
 
 ### Build docker image manually
 
 Please run below command to build manually.
 
 ```bash
 cd /path/to/android-cuttlefish
-cd docker/arm-server
+cd docker/linux-infrastructure-docker
```

Comment:
the `docker` suffix here is redundant as we are within the docker directory.

why the name `linux-infrastructure`? from my understanding, this directory seems related to build a docker image ready to manage cuttlefish instances through out the host orchestrator, wondering whether the full path name: `android-cuttlefish/docker/orchestration` makes sense here?

Reply:
I actually waited for the better naming, as I wrote in the other comment. There's no huge meaning for setting `linux-infrastructure`, the primary purpose is to avoid dir name relying on specific architecture.(e.g. arm)

Reply:
Just for clarification, what's your suggestion exactly? Is it changing directory name only, or the docker image name as well? For dir name, I 100% agree. For the image name, I have minor concern that it may cause little confusion to developers to distinguish between host orchestrator, cloud orchestrator, and the docker named with `cuttlefish-orchestration` or similar.

OTOH, I don't hate your suggestion since it sounds better than my naming at least I think. I'll change that in advance.

Reply:
Changed to the `orchestration`.